pkg/lite-apiserver/proxy/proxy.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 package proxy
 
 import (
+	"bytes"
 	"fmt"
+	"io"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
```

Comment:
L23 is outstanding. Maybe we could format it by adding a newline.

Reply:
done. thx